edisgo/grid/network.py

```diff
@@ -2064,6 +2075,62 @@ def grid_expansion_costs(self):
     def grid_expansion_costs(self, total_costs):
         self._grid_expansion_costs = total_costs
 
+    @property
+    def grid_losses(self):
+        """
+        Holds the losses in the grid obtained from the slack bus in
+        kW and kvar.
+
+        Returns
+        -------
+        :pandas:`pandas.DataFrame<dataframe>`
+            Total Losses, both active and reactive power losses
+            per timestep
+        """
+
+        return self._grid_losses
+
+    @grid_losses.setter
+    def grid_losses(self, pypsa_grid_losses):
+        self._grid_losses = pypsa_grid_losses
+
+    @property
+    def grid_exchanges(self):
```

Comment:
I would rather call it slack_results or hv_mv_exchanges?

Reply:
hv_mv_exchanges sounds clearer, the description has been updated to explain that there are no losses considered in this and The correction will be pushed with the PR of branch test/curtailment_results